src/scripts/activate-user.ts

```diff
@@ -37,23 +42,17 @@ export const activateUser = (
       Console.info(
```

Comment:
**Issue: ** This shouldn't happen so low 'down' in the stack. Let's move this logging up into `dryRunActivateUser` (by adding current status to the existing log message you have) and in `activateUser` with a new `Console.info` step.

Reply:
added in commit: 24c1ffb0115047966275f9de038f03259c636f52